MetaMask/metamask-extension
# feat: multi RPC editor

## **Description**

Initial UI for a multi rpc editor in the network form.


https://github.com/MetaMask/metamask-extension/assets/3500406/1b38f697-56e6-4594-8c74-6fd341c01796


[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/25219?quickstart=1)

## **Related issues**


## **Manual testing steps**

- build with yarn; ENABLE_NETWORK_UI_REDESIGN=1 yarn start
- Settings -> networks
- Should see multi rpc dropdown on ethereum network


## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
